apps/api/src/coscrad-cli/ingest-media-items.cli-command.ts

```diff
@@ -29,6 +41,76 @@ interface IngestMediaItemsCliCommandOptions {
     staticAssetDestinationDirectory: string;
 }
 
+const buildCreateResourceFsaForMediaItem = (
+    {
+        aggregateCompositeIdentifier: { id: mediaItemId },
+        title,
+        mimeType,
+        lengthMilliseconds,
+    }: CreateMediaItem,
+    generatedId: AggregateId
+): CommandFSA<ICommandBase> => {
+    if (isNullOrUndefined(generatedId)) {
+        throw new Error(`missing id`);
+    }
+
+    if (isAudioMimeType(mimeType)) {
+        const fsa: CommandFSA<CreateAudioItem> = {
+            type: `CREATE_AUDIO_ITEM`,
+            payload: {
+                aggregateCompositeIdentifier: {
+                    type: AggregateType.audioItem,
+                    id: generatedId,
+                },
+                name: title,
+                // this is the only choice for media item titles currently
+                languageCodeForName: LanguageCode.English,
+                lengthMilliseconds,
+                mediaItemId,
+            },
+        };
+
+        return fsa;
+    }
+
+    if (isVideoMimeType(mimeType)) {
+        const fsa: CommandFSA<CreateVideo> = {
+            type: `CREATE_VIDEO`,
+            payload: {
+                aggregateCompositeIdentifier: {
+                    type: AggregateType.video,
+                    id: generatedId,
+                },
+                name: title,
+                languageCodeForName: LanguageCode.English,
+                lengthMilliseconds,
+                mediaItemId,
+            },
+        };
+
+        return fsa;
+    }
+
+    if (isMimeTypeAllowedForPhotograph(mimeType)) {
+        const fsa: CommandFSA<CreatePhotograph> = {
+            type: `CREATE_PHOTOGRAPH`,
+            payload: {
+                aggregateCompositeIdentifier: {
+                    type: AggregateType.photograph,
+                    id: generatedId,
+                },
+                title,
+                languageCodeForTitle: LanguageCode.English,
+                mediaItemId,
+                // TODO What should we do about this? Maybe it is indicated in the filename? The real metadata? Should be optional?
```

Comment:
could we eventually make a command to add photographer?  Would `photographer` connect to the `creators`?